### PR TITLE
Updates WORKSPACE proxy for submodule testing

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -1,3 +1,7 @@
+PROXY = "eb8bfccc391e528fba316c4e081515eb18583e5c"
+
+PROXY_SHA = ""
```
Comment (kyessenov) [inline WORKSPACE]: is this supposed to be empty?

Comment (sebastienvas) [inline WORKSPACE]: Yes there is no SHA for that particular commit. "" is the default, meaning no SHA.

**Comment by sebastienvas** [comment]
It seems that the last proxy broke the build.

**Comment by kyessenov** [comment]
I moved proxy SHA to PROXY variable so we can close this I think.

**Pull request closed without merging**